CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2.3.1
+
+### Changes
+
+* Avoid statusbar resize when showing progressbar
```

Comment:
I'd even call that a fix. (I noticed weird behavior but didn't look into it 😬)

Reply:
Moved it to bugs fixed